meringue-maven-plugin/src/main/java/edu/neu/ccs/prl/meringue/AnalysisMojo.java

```diff
@@ -103,9 +110,12 @@ public void execute() throws MojoExecutionException {
             getLog().info("Writing coverage report to: " + coverageReportFile);
             getLog().info("Writing failures report to: " + failuresReportFile);
             report.write(coverageReportFile, failuresReportFile);
-            File htmlReportDir = new File(getOutputDir(), "jacoco-report");
-            getLog().info("Writing JaCoCo report to: " + htmlReportDir);
-            report.writeHtmlReport(getTestDescription(), htmlReportDir);
+            File reportDir = new File(getOutputDir(), "jacoco-report");
+            getLog().info("Writing JaCoCo report to: " + reportDir);
+            reportDir.mkdirs();
```

Comment:
`mkdirs` was not in the original code? What directories does it even create? Did you see this from the JaCoCo code?

Reply:
Yes. It creates the `jacoco-report` folder. I added this here mainly because that when generating only a `jacoco.csv` or `jacoco.xml`, this folder is not automatically created like when generating an HTML report.